R/teal_lockfile.R

```diff
@@ -0,0 +1,90 @@
+#' Create `renv` `.lockfile`
+#'
+#' This function is used during `teal::init` and enables to create `renv` `.lockfile` to be shared in the `teal` app.
```

Comment:
```suggestion
#' This function is used during `teal::init` and creates `renv` compatible `.lock` file to used inside the application.
```

Reply:
Hey @pawelru, `renv` just uses lockfile. They do not quote it, nor they put a dot, nor they separate `lock` from `file`. I will rewrite the docs, so it's a regular word, all over the place

Reply:
yes!

Reply:
coolio